.travis.yml

```diff
@@ -68,6 +68,16 @@ jobs:
         - git diff --exit-code
         - docker exec pgbuild /bin/bash -c "/tsdb_build/scripts/export_prefix_check.sh"
 
+    - if: (type = pull_request) OR (type = cron) OR NOT (branch = master)
```

Comment:
Hmm adding this to travis seems a bit redundant when its already a commit hook, i would rather not add this to travis.

Reply:
tests aren't run by the commit hook, fwiw.

Reply:
I guess we could run it in the cron job if you feel strongly about this